networkx/algorithms/centrality/load.py

```diff
@@ -108,91 +110,107 @@ def _node_betweenness(G,source,cutoff=False,normalized=True,weight=None):
+    onodes = [(l, vert) for (vert, l) in length.items()]
     onodes.sort()
-    onodes[:] = [vert for (l,vert) in onodes if l>0]
+    onodes[:] = [vert for (l, vert) in onodes if l > 0]
 
     # intialize betweenness
-    between={}.fromkeys(length,1.0)
+    between = {}.fromkeys(length, 1.0)
 
     while onodes:
-        v=onodes.pop()
+        v = onodes.pop()
         if v in pred:
-            num_paths=len(pred[v])   # Discount betweenness if more than
-            for x in pred[v]:        # one shortest path.
-                if x==source:   # stop if hit source because all remaining v
-                    break       #  also have pred[v]==[source]
-                between[x]+=between[v]/float(num_paths)
+            num_paths = len(pred[v])  # Discount betweenness if more than
+            for x in pred[v]:         # one shortest path.
+                if x == source:  # stop if hit source because all remaining v
+                    break        # also have pred[v]==[source]
+                between[x] += between[v] / float(num_paths)
     #  remove source
     for v in between:
-        between[v]-=1
+        between[v] -= 1
     # rescale to be between 0 and 1
     if normalized:
-        l=len(between)
+        l = len(between)
         if l > 2:
-            scale=1.0/float((l-1)*(l-2)) # 1/the number of possible paths
+            # scale by 1/the number of possible paths
+            scale = 1.0 / float((l - 1) * (l - 2))
             for v in between:
                 between[v] *= scale
     return between
 
 
-load_centrality=newman_betweenness_centrality
+load_centrality = newman_betweenness_centrality
 
 
-def edge_load(G,nodes=None,cutoff=False):
+def edge_load(G, cutoff=False):
     """Compute edge load.
 
-    WARNING:
+    WARNING: This concept of edge load has not been analysed
+    or discussed outside of NetworkX that we know of.
+    It is based loosely on load_centrality in the sense that
+    it counts the number of shortest paths which cross each edge.
+    This function is for demonstration and testing purposes.
+
+    Parameters
+    ----------
+    G : graph
+        A networkx graph
 
-    This module is for demonstration and testing purposes.
+    cutoff : bool, optional
+        If specified, only consider paths of length <= cutoff.
 
+    Returns
+    -------
+    A dict keyed by edge 2-tuple to the number of shortest paths
+    which use that edge. Where more than one path is shortest
+    the count is divided equally among paths.
     """
-    betweenness={}
-    if not nodes:         # find betweenness for every node  in graph
-        nodes = list(G)   # that probably is what you want...
-    for source in nodes:
-        ubetween=_edge_betweenness(G,source,nodes,cutoff=cutoff)
-        for v in ubetween.keys():
-            b=betweenness.setdefault(v,0)  # get or set default
-            betweenness[v]=ubetween[v]+b    # cumulative total
+    betweenness = {}
```

Comment:
This could be `betweenness = defaultdict(int)`, then below just `betweenness[e] += ubetweenv`. Finally, cast to `dict` when returning.